projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html

```diff
@@ -0,0 +1,174 @@
+<h1 class="govuk-heading-xl" id="Accessibility-statement-for-Expert-UI">Accessibility statement for Expert UI</h1>
+
+<ul class="govuk-list">
+  <li><a href="accessibility#immigration">Immigration and Asylum</a></li>
+  <li><a href="accessibility#family">Family Public Law </a></li>
+</ul>
+
+<p class="govuk-body">This website is run by HM Courts & Tribunals Service. We want as many people as possible to be
+  able to use this website. For example, you should be able to:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>change colours, contrast levels and fonts</li>
+  <li>zoom in up to 300% without the text spilling off the screen</li>
+  <li>navigate most of the website using just a keyboard</li>
+  <li>navigate most of the website using speech recognition software</li>
+  <li>listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and
+    VoiceOver)
+  </li>
+</ul>
+
+<p class="govuk-body">We’ve also made the website text as simple as possible to understand.</p>
+
+<p class="govuk-body"><a rel="noopener noreferrer" target="_blank" href="https://mcmw.abilitynet.org.uk/">AbilityNet</a>
+  has advice on making your device easier to use if you have a disability.</p>
+
+<h2 class="govuk-heading-l" id="How-accessible-this-website-is">How accessible this website is</h2>
+
+<p class="govuk-body">We know some parts of the website are not fully accessible:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>some pages are difficult to navigate using just a keyboard</li>
+  <li>some pages can’t be listened to in full using a screen reader</li>
+  <li>some pages have problems with colour contrasts between text and background</li>
+  <li>some PDFs may not be fully accessible to a screen reader</li>
+</ul>
+
+<h2 class="govuk-heading-l" id="What-to-do-if-you-cannot-access-parts-of-this-website">Feedback and contact
+  information</h2>
+
+<p class="govuk-body">If you need information on this website in a different format like accessible PDF, large print,
+  easy read, audio recording or braille, you can:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>email: <a href="mailto:#"> HMCTSforms@justice.gov.uk</a></li>
+  <li>call: +44 (0) 300 123 1711</li>
```

Comment:
just also wondering if this should have a <a href="tel:"> ...

Reply:
The requirement doesn't need a a for call, so will leave it